src/App.js

```diff
@@ -1,24 +1,32 @@
 import './App.css';
 import '@fontsource/roboto';
 import React from 'react';
-import Typography from '@material-ui/core/Typography';
-import Button from '@material-ui/core/Button';
-import { Container } from '@material-ui/core';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route
+} from "react-router-dom";
+import Home from './components/pages/Home'
+import DrsIndex from './components/pages/drs/DrsIndex';
+import DrsShow from './components/pages/drs/DrsShow';
 
 class App extends React.Component {
   render(){
     return (
-      <div className="App">
-        <meta
-          name="viewport"
-          content="minimum-scale=1, initial-scale=1, width=device-width"
-        />
-        <Container maxWidth="lg">
-          <Typography variant="h1" gutterBottom>Welcome to the GA4GH Starter Kit</Typography>
-          <Typography variant="h3" gutterBottom>Get Started</Typography>
-          <Typography variant="body1" gutterBottom>Click the buttons below to start using one of the GA4GH Starter Kits</Typography>
-          <Button variant="contained" color="primary" size="large" onClick={() => { }}>DRS Starter Kit</Button>
-        </Container>
+      <div>
+        <Router>
```

Comment:
can we reorder these such that the homepage is at the top, followed by `/drs`, and lastly `/drs/:id`? i.e. similar to the order one would traverse these routes when accessing the app. You may have to add the `exact` prop for each Route, see [this page](https://github.com/ga4gh/cloud-interop-ui/blob/master/src/App.js)